orchestra/management/commands/runstaffbot.py

```diff
@@ -0,0 +1,29 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.utils import autoreload
+from slackclient import SlackClient
+
+from orchestra.bots.staff_bot import StaffBot
+from orchestra.core.errors import SlackConnectionError
+
+
+class Command(BaseCommand):
+    help = 'Runs a Slack StaffBot for RTM calls'
+
+    def handle(self, *args, **options):
+        autoreload.main(self.inner_run, None, options)
+
+    def inner_run(self, *args, **options):
+        slack_client = SlackClient(settings.STAFF_BOT_API_KEY)
```

Comment:
We currently use `slacker` for other Slack API-related work.  Does the RTM class there do what we want?

https://github.com/os/slacker/blob/256b21bc45fda9584a30158360ed9d735b20c53e/slacker/__init__.py#L482